app/src/main/java/org/simple/clinic/newentry/PatientEntryScreenController.kt

```diff
@@ -181,27 +176,4 @@ class PatientEntryScreenController @Inject constructor(
 
     return showErrors.mergeWith(scrollToFirstError)
   }
-
-  private fun savePatient(events: Observable<UiEvent>): Observable<UiChange> {
-    val ongoingEntryChanges = events
-        .ofType<OngoingPatientEntryChanged>()
-        .map { it.entry }
-
-    val canPatientBeSaved = ongoingEntryChanges
```

Comment:
This behaviour of checking validation errors before saving was lost. Please take a look?

Reply:
Fixed.